dcpy/test/conftest.py

```diff
@@ -0,0 +1,34 @@
+import boto3
```

Comment:
is it possible to move this conftest to the `dcpy/test/utils` folder? I ask because we also have integration tests in the `dcpy/test/connectors` folder, and we wouldn't want the fixtures to affect anything there. Wouldn't cause problems with the existing test for github, but would cause an issue for future s3 integration tests.